### ISSUE TITLE
Demo app wrong output for Tensorflow PB converted model

### ISSUE THREAD

**Issue by SanthoshRajendiran** [alibaba/MNN#80]
We tried including softmax operator on top of the current portrait model. We were able to get the output from the graph at the pb and tflite level. When we tried the same with demo app, provided by you, it gave wrong results. 

Expected Softmax Output: Values in the range 0 to 1.
Current Output: Random float values like 0.0 , 4.98, 6.24 etc.,

**Comment by jxt1234** [comment]
Do you delete this code: 
"mConfig.outputTensors[0] = "ResizeBilinear_2";"
In line 89
demo/android/app/src/main/java/com/taobao/android/mnndemo/PortraitActivity.java

**Comment by SanthoshRajendiran** [comment]
Yes.. I replaced the output tensor with Softmax operator that is in my pb file.

**Comment by jxt1234** [comment]
You can run the model by ./MNNV2Basic.out to check the result firstly.
